### ISSUE TITLE
Help with Vertical Monitors

### ISSUE THREAD

**Issue by pjdurai** [fuhsjr00/bug.n#127]
Hi Folks

I am new to bug.n and trying to figure out how do do vertical monitors.

Instead of splitting left half as primary and right half for the stack, can I configure it to have the top half as primary and bottom half as the stack ?

Could someone point me in the right direction?

Appreciate your time.

Thanks
pj

**Comment by x29a** [comment]
Hi,

have you tried various settings for the various axis properties as described in the [documentation](https://github.com/fuhsjr00/bug.n/blob/master/doc/Default_configuration.md#window-arrangement)? Which version of Bug.N are you running and with which windows? Dual Screen setup? Can you share your config?

**Comment by joten** [comment]
Using the hotkey <kbd>Win</kbd><kbd>Ctrl</kbd><kbd>T</kbd> or the following lines in Config.ini (as an example for the second view on the first monitor) should fit the bill:
```
View_#1_#2_layout_#1=1
View_#1_#2_layoutAxis_#1=2
View_#1_#2_layoutAxis_#2=1
View_#1_#2_layoutAxis_#3=1
```

**Comment by chaorace** [comment]
@joten I can't seem to get the given layout rule to work, anything in the Config file that doesn't start with Config_hotkey or Config_rule seems to be ignored. This is using the latest stable release (v9.0.2).

**Comment by joten** [comment]
You are right. `View_#*` is part of the session management, therefor you should be able to set the above in Config.ini, when bug.n is *not* running, or set the layout via the hotkeys (i.a. <kbd>Win</kbd><kbd>Ctrl</kbd><kbd>T</kbd>); with session management it should be the same after a reload of bug.n or Windows re-login.
